src/sentry/remote_config/docs/protocol.md

```diff
@@ -79,11 +127,34 @@ If the server's ETag does not match the request's a 200 response is returned.
       "features": [
         {
           "key": "hello",
-          "value": "world"
+          "value": "world",
```

Comment:
I'm assuming that `"world"` would be returned if either:
A) the conditions below do not match. ie: request contains `?context={region: Asia, now: 0}`
B) the user_id hashes to the wrong side of the rollout, so 50% of the time europeans will see "world"

Reply:
Yes.